demo/app.js

```diff
@@ -156,126 +179,176 @@ class App extends React.Component {
     this.setState({ fileName: value });
   };
 
+  handleAutoSaveChanges = newAutoSaveData => {
+    console.log("handleAutoSaveChanges", newAutoSaveData);
+    const { data, ext } = newAutoSaveData;
+    this.setState({ autoSaveData: data, autoSaveExtension: ext });
```

Comment:
If the autoSaveExtension is stored here (and assuming that's the correct one), would you actually still want autoSaveFormat?

Reply:
Maybe the naming needs to be changed to be more clear `autoSaveExtension` refers to the file extension. eg `.srt` or `.json` file while `autoSaveFormat` is `draftjs` or `digitalpaperedit` etc..  if that makes sense?

Reply:
ah right, yeah that makes sense. ok, if you can - can you quickly rename `autoSaveFormat` to something like `autoSaveObjectType`?